python/sglang/srt/custom_op.py

```diff
@@ -40,6 +41,8 @@ def dispatch_forward(self):
             return self.forward_cuda
         elif _is_hip:
             return self.forward_hip
+        elif not torch.cuda.is_available() and cpu_has_amx_support():
```

Comment:
can we check if the input is run on CPU and do
```
if device == torch.device("cpu"):
  return self.forward_cpu

right now, self.forward_cpu won't be called from dispatch_forward?

and inside `forward_cpu`, we can check wether amx is supported.

Reply:
Now, we use `global_server_args_dict["device"]` instead to determine if it is running on CPU.
self.forward_cpu will not be called, it will run into RMSNorm's forward_cpu.